custom/icds/location_reassignment/dumper.py

```diff
@@ -128,8 +128,10 @@ def _get_rows_for_split(self, details):
 
     @memoized
     def _get_count_of_cases_owned(self, site_code):
-        location_id = self._old_location_ids_by_site_code()[site_code]
-        return len(self.case_accessor.get_case_ids_by_owners([location_id]))
+        location_id = self._old_location_ids_by_site_code().get(site_code)
+        if location_id:
+            return len(self.case_accessor.get_case_ids_by_owners([location_id]))
+        return "Not Found"
```

Comment:
Why not return 0, so that the return value is always a number?

Reply:
0 would be returned if there are no assigned cases, so wanted to flag this separately.